credentials/alts/alts_test.go

```diff
@@ -300,3 +359,49 @@ func versions(minMajor, minMinor, maxMajor, maxMinor uint32) *altspb.RpcProtocol
 		MaxRpcVersion: version(maxMajor, maxMinor),
 	}
 }
+
+func startFakeHandshakerService(t *testing.T, wait *sync.WaitGroup) (stop func(), address string) {
+	listener, err := testutils.LocalTCPListener()
+	if err != nil {
+		t.Fatalf("LocalTCPListener() failed: %v", err)
+	}
+	s := grpc.NewServer()
+	altspb.RegisterHandshakerServiceServer(s, &testutil.FakeHandshaker{})
```

Comment:
This should be `altsgrpc` with a separate import.

Reply:
Done, sorry I missed this one. :)